UnityProject/Assets/Scripts/Player/PlayerHealthData.cs

```diff
@@ -39,6 +39,8 @@ public class RaceHealthData
 	public BloodType BloodType;
 
 	public ImplantProcedure RootImplantProcedure;
+
+	public List<string> BodyPartsThatShareTheSkinTone = new List<string>();
```

Comment:
Wouldn't be easyer to spsfiy on the body part it self?

Reply:
Wdym? you're already specifying what body parts to use here.

Reply:
I got what you meant just now and made changes, now you can assign the body part instead of a string.